Makefile

```diff
@@ -4,3 +4,11 @@ all:
 .PHONY: test
 test:
 	python3 -m pytest
+
+.PHONY: checkstyle
+checkstyle:
+	black --check ./
+
+.PHONY: tidy
+tidy:
+	black ./
```

Comment:
I suggest to use the same style as in
https://github.com/os-autoinst/openQA/blob/master/Makefile#L139= , e.g. include style checks as part of test